runtime/syntax/sml.vim

```diff
@@ -4,12 +4,18 @@
 " Maintainers:  Markus Mottl            <markus.mottl@gmail.com>
 "               Fabrizio Zeno Cornelli  <zeno@filibusta.crema.unimi.it>
 " URL:          http://www.ocaml.info/vim/syntax/sml.vim
```

Comment:
This link is broken. It says "Not Found. The requested URL /vim/syntax/sml.vim was not found on this server."

Reply:
Thanks, I have removed the URL.  I do not intend to maintain a separate location in the future.